### PR TITLE
Remove private server fix

### PR DESCRIPTION
**Pull request by TooAngel**
This room reset was introduced as a bug on the private server.

### CODE REVIEW

**Comment by TooAngel** [comment]
DCBOT: 
This repository is under [democratic collaboration](https://github.com/TooAngel/democratic-collaboration) and will be merged automatically.

The merge decision is based on the outcome of the reviews:
 - `Approve` add the reviewer value (number of commits) to the `metric`
 - `Request changes` substract the reviewer value from the `metric`

 The merge will happen after `(1 - metric/total.votes) * (5 + pull_request.commits * 5))` calculated from the last code change.

Please review the PR to make a good democratic decision.

Summoning some reviewers:
 - @Nevrdid: A proper review is appreciated
 - @WhiteHalmos: I always like your reviews
 - @Somotaw: I always like your reviews

File: src/prototype_room_controller.js [hunk 1]
```diff
@@ -16,12 +16,6 @@ Room.prototype.buildBase = function() {
         this.setup();
       }
     }
-    // FIX for a server bug, where exits were not available in non standard rooms (fixing the bots on these servers)
-    resetCounters(this);
```
Comment (Somotaw) [inline src/prototype_room_controller.js]: was this supposed to be removed? :)

Comment (TooAngel) [inline src/prototype_room_controller.js]: good catch, thx

**Comment by democratic-collaboration** [comment]
DCBOT: A new review, yeah.

    Votes: 179/299
    Coefficient: 0.598662207358
    Merging in 4 days -1 hours
    Age 0 days 0 hours

**Comment by democratic-collaboration** [comment]
DCBOT: A new review, yeah.

    Votes: 231/302
    Coefficient: 0.764900662252
    Merging in 1 days 5 hours
    Age 1 days 2 hours

**Comment by democratic-collaboration** [comment]
DCBOT: A new review, yeah.

    Votes: 242/306
    Coefficient: 0.790849673203
    Merging in 1 days -12 hours
    Age 1 days 13 hours

**Pull request merged**